src/xrGame/PHCommander.h

```diff
@@ -82,13 +83,16 @@ class CPHCommander : public IPHWorldUpdateCallbck
         CPHCondition* condition, CPHReqComparerV* cmp_condition, CPHAction* action, CPHReqComparerV* cmp_action);
     void add_call(CPHCondition* condition, CPHAction* action);
     void add_call_threadsafety(CPHCondition* condition, CPHAction* action);
+    void AddCallDeferred(CPHCondition* condition, CPHAction* action);
```

Comment:
Is it possible to add 'const' modifier to the arguments?

Reply:
No, because next stage wiil be assigning of 'condition'/'action' to 'm_condition'/'m_action'. And entities of type 'const CPHCondition*' can't be assigned to an entity of type 'CPHCondition*'. Therefore, we'll have to use 'const_cast'.